account_move_name_sequence/readme/DESCRIPTION.rst

```diff
@@ -7,6 +7,75 @@ In Odoo version 14.0, the number of journal entries can be manually set by the u
+
+In Odoo version 14.0, the number of journal entries can be manually set by the user. Then, the number attributed for next journal entries in the same journal is computed by a complex piece of code that guesses the format of the journal entry number from the number of the journal entry which was manually entered by the user. It has several drawbacks:
+
+* the available options for the sequence are limited,
+* it is not possible to configure the sequence in advance before the deployment in production,
+* as it is error-prone, they added a *Resequence* wizard to re-generate the journal entry numbers, which can be considered as illegal in many countries,
+* the `piece of code <https://github.com/odoo/odoo/blob/14.0/addons/account/models/sequence_mixin.py>`_ that handle this is not easy to understand and quite difficult to debug.
+
+Odoo>=v14.0 raises new concurrency issues since it locks the last journal entry of the journal to get the new number causing a bottleneck
+Even if you only are creating a draft journal entry it locks the last one
+It applies to all accounting Journal Entries
+
+e.g.
+
+ - Customer Invoices,
+ - Credit Notes
+ - Customer Payments
+ - Vendor Bills
+ - Vendor Refunds
+ - Vendor Payment
+ - Manual Journal Entries
+
+Even if your country or company allows relax gaps in a few of these kinds of documents (e.g. only Payments) you are not able to change the implementation to standard allowing gaps of the sequence anymore
+So, all the time it is using no-gap for all journals without an extra table to sequence but locking the last Journal Entry record
+
+Then, the following concurrency errors are being raised now frequently:
+
+* Editing the last record used to get the new number from other process
+* Creating a new draft invoice/payment (not only when posting it)
+* Creating a transaction to create invoice then payment or vice versa it raises a deadlock error
+* Reconciling last record since that it could be a heavy process
+* Creating 2 or more Invoices/Bills at the same time
+* Creating 2 or more Payments at the same time
+* Creating 2 or more Journal Entries at the same time
+
+
+All these increases on concurrency errors bring more issues since that Odoo is not prepared:
+
+* Using e-commerce, configured with Invoicing Policy Ordered and Automatic Invoice, the portal users will see errors in the checkout even if the payment was done, the sale order could be in draft state and request a new payment, so double charges
+* Using subscription, auto creating invoice after payment you see subscription in payment exception
+* Using accounting creating invoice or payment, you will see errors then you will need to start the process again and again until you get the lock before than another user
+* The workers could be used for more time than before since that it could be waiting for release so less concurrent users supported or loading page shown more frequently affecting the performance
```

Comment:
```suggestion
* The workers could be used for more time than before since that it could be waiting for release so less concurrent users supported or loading page is shown more frequently affecting the performance
```